packages/icons/icons.yml

```diff
@@ -5903,22 +6037,39 @@
     - 32
 - name: forecast--hail
   friendly_name: forecast--hail
```

Comment:
@LMapes none of the forecasts look like friendly names can you please confirm: 

- [ ]  `friendly_name: forecast--hail`
- [ ]  `friendly_name: forecast--hail-30`
- [ ] `friendly_name: forecast--lightning`
- [ ]  ` friendly_name: forecast--lightning-30`
- [ ] `friendly_name: forecast--lightning-30`

- [ ] `  friendly_name: ice--accretion`

Reply:
Forecast hail
Forecast hail 30
Forecast lightning
Forecast lightning 30
Forecast lightning 60 ??? Not sure about this, is a duplicate or is there a 60 as well?
Ice accretion